src/components/sections/MajorFeatures.jsx

```diff
@@ -36,7 +36,7 @@ class MajorFeatures extends React.Component {
             </div>
             <H2>Explore</H2>
             <div className={s.divider}></div>
-            <Text>Check out our library of legal templates generated by our fast-growing community.</Text>
+            <Text><a href="https://app.openlaw.io/templates"><strong>Check out</strong></a> our library of legal templates generated by our fast-growing community.</Text>
```

Comment:
Nice. One thing: I think we could be even more contextual and use the "legal templates" as the link text, instead of "check out".

Reply:
That sounds good!